sass/woocommerce/_archives.scss

```diff
@@ -83,7 +83,7 @@
 				text-align: center;
 
 				@media (max-width: 768px) {
-					& {
+					@at-root body.responsive#{&} {
```

Comment:
Hope that works. 

FYI: It works with [theme-build](https://github.com/siteorigin/theme-build) :)